news/7815.bugfix

```diff
@@ -0,0 +1 @@
+This change will switch from using full file path for wheel to using just the name of the package.
```

Comment:
This is supposed to be a line item in https://pip.pypa.io/en/latest/news/, so it'd be better to re-pharse this as:

```suggestion
Display only the wheel filename instead of the entire filepath, when using a cached wheel.
```